.github/workflows/monty.yml

```diff
@@ -277,6 +330,52 @@ jobs:
           path: |
             ~/miniconda
           key: ${{ steps.restore_cache.outputs.cache-primary-key }}
+          
+  should_run_docs:
```

Comment:
**issue**: By our convention, this name should end with `_monty`, and the job name below should be `name: should-run-docs-monty`.